### PR TITLE
FIF-238 Implement mockup style on Teacher Landing page with styled-components

### PR DESCRIPTION
**Pull request by kurtjohnson-HS**
When applied this task should apply the styled-components React library to provide CSS-in-JS.

The mocks are on [InvisionApp](https://projects.invisionapp.com/share/BVXR5QM5QJF#/screens) and [Zeplin.io](https://app.zeplin.io/project/5ef23ca769d0fe299e68b291).

### CODE REVIEW

**Comment by stephenfuqua** [comment]
1. Is it possible to load the WorkSans font from CDN instead of having the files directly in our repository?
2. I need to study the Open Font License and possibly check with legal team before we commit to using or distributing this font.
2. What is the source for the image files? Do we have the proper license to re-distribute them?

**Comment by NSD-Xavier** [comment]
Maybe we can use something like @import url('https://fonts.googleapis.com/css2?family=Work+Sans:ital@1&display=swap'); in css, or <link href="https://fonts.googleapis.com/css2?family=Work+Sans:ital@1&display=swap" rel="stylesheet"> in html

**Comment by kurtjohnson-HS** [comment]
> 1. Is it possible to load the WorkSans font from CDN instead of having the files directly in our repository?
Yes, I will make that change.
> 2. I need to study the Open Font License and possibly check with legal team before we commit to using or distributing this font.
Ok.,
> 3. What is the source for the image files? Do we have the proper license to re-distribute them?
Dripicons.  [Creative Commons](https://github.com/amitjakhu/dripicons/blob/master/LICENSE) is the license used.

**Comment by stephenfuqua** [comment]
In the DripIcons readme the developer wrote "No attribution or backlinks are required, but any form of spreading the word is always appreciated!". However, he also used a licenses that plainly states that attribution is required (images are CC-attribution-sharealike). 😆 

https://creativecommons.org/licenses/by-sa/4.0/

> Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.
>
> ShareAlike — If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original.

So the notices.md file should have something like...

```
## Dripicons

Copyright (c) Amit Jakhu, some rights reserved. 
Freely used without modification under terms of the [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/) 
license; please see the [full license](https://creativecommons.org/licenses/by-sa/4.0/legalcode) for 
warranty disclaimer notifications. [Original source: amitjakhu/dripicons](https://github.com/amitjakhu/dripicons).
```

File: NOTICES.md [hunk 1]
```diff
@@ -10,3 +10,105 @@ open source packages that are attributed below in this notice. Where required,
...
+4.0](https://creativecommons.org/licenses/by-sa/4.0/) license; please see the
+[full license](https://creativecommons.org/licenses/by-sa/4.0/legalcode) for
+warranty disclaimer notifications. [Original source:
+amitjakhu/dripicons](https://github.com/amitjakhu/dripicons).
+
+## Work Sans Font
+
+https://github.com/weiweihuanghuang/Work-Sans
+
+Copyright (c) 2019, The Work Sans Project Authors
```
Comment (stephenfuqua) [inline NOTICES.md:29]: Not absolutely critical, but might make for nicer formatting: can you add a tab (4 spaces) in front of each of lines 29 through 114?

File: EdFi.Buzz.UI.Angular/src/app/Components/StudentCardReact/studentCard.tsx [hunk 2]
```diff
@@ -2,7 +2,158 @@ import { Student } from 'src/app/Models';
 import { useState } from 'react';
 import * as React from 'react';
 
-import './studentCard.css';
+import './StudentCard.css';
```
Comment (NSD-Xavier) [inline EdFi.Buzz.UI.Angular/src/app/Components/StudentCardReact/studentCard.tsx:5]:
```suggestion
import './studentCard.css';
```
If case are different I get 
WARNING in ./src/app/Components/StudentCardReact/StudentCard.css 
There are multiple modules with names that only differ in casing.

**Pull request merged**